packages/mux-uploader-react/src/common/utils.ts

```diff
@@ -0,0 +1,54 @@
+// NOTE: As a forward-looking implementation, we may want to assume
```

Comment:
thought(non-blocking): We should consider eventually moving this into its own "shared" package to avoid the redundant code.